web-api.env

```diff
@@ -1,8 +1,8 @@
 # Wep API server settings
 PORT=5000
 FORCEHTTPS=false
-RSA_PASSPHRASE=PASTE_RSA_SECRET
-APPLICATION_LICENSE_KEY=PASTE_LICENSE_KEY
+RSA_PASSPHRASE=aAbBcCdDeEfF
+APPLICATION_LICENSE_KEY=A2CE5C-F4A70D-47FF85-261A3B-ACF503-8685C6
```

Comment:
why has this file been changed? this leakes the license key and secret in a public repo